proto/connectrpc/conformance/v1alpha1/config.proto

```diff
@@ -19,35 +19,59 @@ package connectrpc.conformance.v1alpha1;
 // Config defines the configuration for running conformance tests.
 // This enumerates all of the "flavors" of the test suite to run.
 message Config {
+  // The features supported by the client or server under test.
+  // This is used to filter the set of test cases that are run.
+  // If absent, an empty message is used. See Features for more
+  // on how empty/absent fields are interpreted.
   Features features = 1;
+  // This can indicate additional permutations that are supported
+  // that might otherwise be excluded based on the above features.
   repeated ConfigCase include_cases = 2;
+  // This can indicates permutations that are not supported even
+  // though their support might be implied by the above features.
   repeated ConfigCase exclude_cases = 3;
 }
 
 message Features {
+  // If empty, HTTP 1.1 and HTTP/2 are assumed.
   repeated HTTPVersion versions = 1;
+  // If empty, all three are assumed: Connect, gRPC, and gRPC-Web.
   repeated Protocol protocols = 2;
+  // If empty, "proto" and "json" are assumed.
   repeated Codec codecs = 3;
-  repeated Compression compression_algorithms = 4;
-  bool supports_h2c = 5;
-  bool supports_tls = 6;
-  bool supports_trailers = 7;
-  bool supports_connect_get = 8;
-  bool requires_connect_version_header = 9;
-  // If non-empty, the set of stream types that are supported. This is
-  // usually for clients, since some client environments may not be
-  // able to support certain kinds of streaming operations, especially
+  // If empty, "gzip" alone is assumed.
```

Comment:
I assume this is because a feature implies all possible compression algos that a client/server could support and since gzip is available by default, it makes sense to assume that. Just wanted to confirm since for the `ClientCompatRequest`, an unspecified compression implies `identity`.

Reply:
Yep, my bad. I'll revise this to indicate that if empty, both "identifty" and "gzip" are assumed.